app/views/lings/_depth_table.html.haml

```diff
@@ -17,7 +17,7 @@
       .pull-right
         %a{:href => "#", :rel => "tooltip", :title => "This is a measure of how many properties have been set for the given language", :data => {:toggle => "tooltip", :animation => "true"} }
           = icon 'question-circle', '', :class => "fa-15x"
-    - if can_see_some?
+    - if can_see_header? [:update, :destroy], lings
```

Comment:
In this case you need to have update and destroy permission to see the action tab